p2p/src/peer.rs

```diff
@@ -39,7 +39,7 @@ pub struct Peer {
 	pub info: PeerInfo,
 	proto: Box<Protocol>,
 	state: Arc<RwLock<State>>,
-	banned_until: Arc<RwLock<time::Tm>>,
+	banned_until: Arc<RwLock<i64>>,
```

Comment:
Maybe better use a RefCell https://doc.rust-lang.org/core/cell/struct.RefCell.html

Reply:
(switched to RefCell and it seems I got persistence to work, yay)